### PR TITLE
[MINOR] EDA-1436 FAQ view

### PR DESCRIPTION
**Pull request by ibrizuela-eb**
Adds frequently asked questions to the Web.

### CODE REVIEW

File: auth_app/templates/auth_app/faq.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                <a class="card-header" href="#" data-toggle="collapse" data-target="#answerSeven"
+                    aria-expanded="true" aria-controls="answerSeven">
+                    <span>As regards EDA selection: If I use Python, will I improve my chances of being selected</span>
+                </a>
+                <div id="answerSeven" class="collapse" aria-labelledby="headingTwo" data-parent="#FaqBar">
+                    <div class="bg-white py-2 collapse-inner rounded ml-3">
+                        <p class="collapse-item">
+                            <strong>No.</strong> You can use the language you like most.<br>
+                            If you are finally hired, the EDA training includes Python.<br>
+                            We will evaluate your code or your solution no matter what language you have used.
```
Comment (vgatica-eb) [inline auth_app/templates/auth_app/faq.html:106]: nit : I know we don´t know a lot about performance and frontend, anyway we could think about refactor this in a future?

File: static/css/sb-admin-2.css [hunk 2]
```diff
@@ -12011,3 +12011,23 @@ footer.sticky-footer .copyright {
...
   width: 100%;
 }
+
+.card-header {
+    background-color: rgb(245,245,245,0.3)!important;
+  border-radius:0!important;
+}
+
+.card{
+border-radius:0!important;
```
Comment (vgatica-eb) [inline static/css/sb-admin-2.css:12021]: nit: why are you using important! here?

File: auth_app/urls.py [hunk 3]
```diff
@@ -17,5 +17,6 @@
     ),
     path('register/', Registration.as_view(), name='register'),
 
-    path('profile/<pk>', Profile.as_view(template_name='auth_app/profile.html'), name='profile'),
+    path('profile/<pk>/', Profile.as_view(template_name='auth_app/profile.html'), name='profile'),
```
Comment (vgatica-eb) [inline auth_app/urls.py:20]: Only a question: why did you change it?

**Review by vgatica-eb — APPROVED** [review]

File: auth_app/templates/auth_app/faq.html (hunk 1 above)
Comment (ibrizuela-eb) [reply auth_app/templates/auth_app/faq.html:106]: Yes. The correct way to solve this is by creating questions and answers in databases. Then you can display as many as you want in the template.

File: static/css/sb-admin-2.css (hunk 2 above)
Comment (ibrizuela-eb) [reply static/css/sb-admin-2.css:12021]: To be sure that the component that is using this class overrides all styles that can overlap with this one.

File: auth_app/urls.py (hunk 3 above)
Comment (ibrizuela-eb) [reply auth_app/urls.py:20]: URL in Django must have a `/` at the end for a convention.
